test/util/utils.test.js

```diff
@@ -0,0 +1,114 @@
+// Copyright IBM Corp. 2013,2016. All Rights Reserved.
+// Node module: loopback
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+var utils = require('../lib/utils');
+var assert = require('assert');
+
+describe('Utils', function() {
+  describe('uploadInChunks', function() {
+    it('should call process function with the chunked arrays', function(done) {
```

Comment:
We don't use `should` in our test names, see http://loopback.io/doc/en/contrib/style-guide.html#test-naming

```js
it('calls process function for each chunk', function(done) {
  // ...
});
```